dplython/dplython.py

```diff
@@ -88,7 +88,8 @@ def apply_on_groups(self, delayedFcn):
     # Drop all 0 index, created by summarize
     if (outDf.index == 0).all():
       outDf.reset_index(drop=True, inplace=True)
-
+    if 'summarize' in str(delayedFcn).lower():
```

Comment:
I'm worried about this check-- if someone else has a function with the text `summarize` in it, it will also trigger this. So it's probably safer to turn `summarize` into a `Verb` and use that as a check.